org/addReviewer.ts

```diff
@@ -0,0 +1,49 @@
+import { danger, peril } from "danger"
+import { PRReviewMetadata } from "../tasks/prReviewReminder"
+import { PullRequest } from "github-webhook-event-types"
+import { PullRequestPullRequestPullRequestUser as User } from "github-webhook-event-types/source/PullRequest"
+
+// This interface gets around the fact that github-webhook-event-types doesn't include a requested_reviewer field
+// on pull_request.review_requested event types (requested_reviewer only exists on certain events).
+// See the GitHub API documentation here: https://developer.github.com/v3/activity/events/types/#webhook-payload-example-28
```

Comment:
I like your thinking! The types are generated automatically from GitHub's API docs [in this script](https://github.com/orta/github-webhook-event-types/blob/master/scripts/generate_types.ts), so that's what we would have to change. It seemed pretty out-of-scope for this change, so I Matt and I thought it'd be better to use this approach instead. Thoughts?

Reply:
Yep! Sounds good for now! Maybe we can create an issue to port the update upstream?

Reply:
sounds good to me - I'm hoping that it's just a matter of running the script Ash mentioned and PRing the results. I just took a few minutes to try it out and had some difficulties with the script out of the box 😅so I think merging this for now and following up would make the most sense

Reply:
Yup, sounds good. Merging as-is makes sense, too, since we'll have production data showing that the change in fact works (not that Orta wouldn't believe us, but "this works for us in production" always a helpful data point when sending a PR to an OSS project).